test/e2e/new-dns-record.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { test, expect } from '@playwright/test';
+
+test.describe('not authenticated', () => {
+  test('redirects to login page', async ({ page }) => {
+    await page.goto('/');
+    await expect(page).toHaveURL(/.*login.*/);
+  });
+});
+
+test.describe('authenticated as user', () => {
+  // Use shared state for being logged in as user
+  test.use({ storageState: 'test/e2e/.auth/user.json' });
```

Comment:
Because every "already logged in" test is going to have to do this, should we create `test/e2e/utils.ts` and make a function, `alreadyLoggedIn()` (or something) that does this for you?

Also, a question: does this need to go in a `beforeAll()` so the order is never an issue?

Reply:
It sets that option for the entire describe block. Actually, using `test.use` in a `beforeEach` or `beforeAll` would result in an error ([relevant docs](https://playwright.dev/docs/api/class-test#test-use))

As for the function, we could create one but all we need is one line per test file I believe. Not sure if that is worth it. 
Would the function simply return this line?

Reply:
I think the value of the function over this is that it can better communicate what is happening here, hide the specifics of the file path, etc.  It's not critical, but if it was me, I'd wrap a named function around this so that others can use it without understanding it.  Up to you.

Reply:
Yea, that makes sense